src/agent/src/image_rpc.rs

```diff
@@ -121,10 +140,44 @@ impl ImageService {
         Ok(())
     }
 
+    // If we fail to start the AA, Skopeo/ocicrypt won't be able to unwrap keys
+    // and container decryption will fail.
+    fn init_attestation_agent() {
+        let config_path = OCICRYPT_CONFIG_PATH;
+
+        // The image will need to be encrypted using a keyprovider
+        // that has the same name (at least according to the config).
+        let ocicrypt_config = serde_json::json!({
+            "key-providers": {
+                "attestation-agent":{
+                    "grpc":AA_PORT
+                }
+            }
+        });
+
+        let mut config_file = fs::File::create(config_path).unwrap();
+        config_file
+            .write_all(ocicrypt_config.to_string().as_bytes())
+            .unwrap();
+
+        // The Attestation Agent will run for the duration of the guest.
+        Command::new(AA_PATH)
+            .arg("--grpc_sock")
+            .arg(AA_PORT)
+            .spawn()
+            .unwrap();
+
+        //TODO: Do we need to clean up the AA on shutdown?
```

Comment:
@fitzthum do you want to keep this a TODO for this PR?

Reply:
I can remove it given that it is not really significant for CCv0. I think there is some discussion to be had about the best way to close persistent connections that the AA has started. If we just use the AA for getting container secrets all at once and at the beginning of execution, then we can probably just have the server timeout and not worry about cleaning anything up for the AA.